deployment/vm_operation.sh

```diff
@@ -67,6 +67,58 @@ power_action() #id_string action(on/off/reset)
     done
 }
 
+snapshot_action()
+{
+    action=$1
+    duration=$2
+    shift 2
+    for id in $@
+    do
+        snapshot_count=`vim-cmd vmsvc/snapshot.get $id | grep "Snapshot Id"| wc -l`
+        vm_name=`vm_getname $id`
+
+        if [ $snapshot_count -gt 1 ]; then
```

Comment:
1. If we use this book system, then global updates won't be done if any vmslave is booked. If updates is necessary, we should disscuss with the developer or we regard this booked slave as an exception, we can update it after booking cancel.

2. In my proposalthere won't be more than one snapshot of a vm.
If so something must go wrong and throw error at the first time is necessary.
Use latest snapshot  may cover such issues.